mne/epochs.py

```diff
@@ -842,8 +842,19 @@ def plot_psd(self, fmin=0, fmax=np.inf, proj=False, n_fft=256,
             End frequency to consider.
         proj : bool
             Apply projection.
-        n_fft : int
-            Number of points to use in Welch FFT calculations.
```

Comment:
I thought that either @Eric89GXL or @agramfort had suggested to treat this as a bug, and to just switch it over to using multitaper instead of Welch, so I didn't think I needed to deprecate. To be able to implement them both, I guess I could add a flag for "kind" where you could supply either 'welch' or 'multitaper', but this might complicate things.